go/worker/common/committee/keymanager.go

```diff
@@ -337,6 +337,12 @@ func (nt *nodeTracker) Nodes(nodes []signature.PublicKey) map[core.PeerID]signat
 }
 
 func (nt *nodeTracker) trackKeymanagerNodes(ctx context.Context) {
+	select {
+	case <-nt.consensus.Synced():
```

Comment:
The consensus needs to be synced so that `WatchStatuses` method doesn't fail when the status notifier hook is executed on subscription.